tabpy-server/tabpy_server/common/default.conf

```diff
@@ -1,8 +1,8 @@
 [TabPy]
 TABPY_QUERY_OBJECT_PATH = /tmp/query_objects
 TABPY_PORT = 9004
-TABPY_STATE_PATH = ./
-#TABPY_BIND_IP = 127.0.0.1
+TABPY_STATE_PATH = ./tabpy-server/tabpy_server
+#TABPY_PWD_FILE = /path/to/password/file.txt
```

Comment:
Should we add a comment that to enable auth to uncomment this line?  We have that suggestion for https support...

Reply:
It is in documentation as there more steps needed (creating passwords file specifically) then just uncommenting the line.

Adding comment anyway...